sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -1,7 +1,16 @@
 # Release History
 
 ## 3.1.0-beta.1 (Unreleased)
+### New Features
+- Added implementation support to create a composed model from the `FormTrainingClient` by calling method `beginCreateComposedModel`.
+- Added properties `modelName` and `customFormModelProperties` to types `CustomFormModel` and `CustomFormModelInfo`.
+- Added property `modelName` to `TrainingOptions` and new type `CreateComposedModelOptions`.
+- Added property `modelId` to `CustomFormSubmodel` and `TrainingDocumentInfo`.
+- Added properties `modelId` and `formTypeConfidence` to `RecognizedForm`. 
+`modelId` has a null value on `RecognizedForm` for prebuilt APIs.
```

Comment:
not sure if we should have this here or in the documentation. 
The same will be for `formTypeConfidence` for models without labels

Reply:
Did we include it in the .NET documentation for receipt API's where we have mentioned about the formType and formTypeConfidence that I can look up for reference.
I am happy to remove it :)

Reply:
I did not https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md#310-beta1-unreleased
I did mention it in the docstrings though https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizedForm.cs#L77

Reply:
Cool, already covered here https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/RecognizedForm.java#L105